src/Slider.jsx

```diff
@@ -159,123 +182,118 @@ class Rheostat extends React.Component {
     this.slideTo = this.slideTo.bind(this);
     this.updateNewValues = this.updateNewValues.bind(this);
     this.setRef = this.setRef.bind(this);
-    this.invalidatePitStyleCache = this.invalidatePitStyleCache.bind(this);
+    this.setHandleNode = this.setHandleNode.bind(this);
+    this.setHandleContainerNode = this.setHandleContainerNode.bind(this);
+    this.positionPercent = this.positionPercent.bind(this);
+  }
 
-    this.pitStyleCache = {};
+  componentDidMount() {
+    // Note: This occurs in a timeout because styles need to be applied first
+    this.handleDimensionsTimeout = setTimeout(() => {
+      this.handleDimensionsTimeout = null;
+      this.setState({ handleDimensions: this.getHandleDimensions() });
+    }, 0);
   }
 
   componentWillReceiveProps(nextProps) {
     const {
-      className,
       disabled,
       min,
       max,
-      orientation,
-      pitPoints,
-      algorithm,
     } = this.props;
+
     const {
       values,
       slidingIndex,
     } = this.state;
 
-    const minMaxChanged = (nextProps.min !== min || nextProps.max !== max);
+    const minMaxChanged = (
+      nextProps.min !== min || nextProps.max !== max
+    );
```

Comment:
this can probs remain unchanged (i think it's less readable on two lines). the parens are probs also unnecessary.